changelog/fragments/1667571017-Add-support-for-running-the-elastic-agent-shipper.yaml

```
@@ -0,0 +1,31 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: Add support for running the elastic-agent-shipper
```

```suggestion
summary: Add experimental support for running the elastic-agent-shipper
```

Let's include the word experimental to make it clear it isn't officially supported as a feature yet even if it is in the code.

---

Done!